daemon/daemon.go

```diff
@@ -62,10 +63,10 @@ import (
 	"github.com/sirupsen/logrus"
 	"go.etcd.io/bbolt"
 	"golang.org/x/sync/semaphore"
-	"golang.org/x/sync/singleflight"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/backoff"
 	"google.golang.org/grpc/credentials/insecure"
+	ctxsingleflight "resenje.org/singleflight"
```

Comment:
[Avoid renaming imports except to avoid a name collision](https://github.com/golang/go/wiki/CodeReviewComments#imports)

Reply:
Yeah I wasn't sure about it. My intent here is to make it explicitly visible, that this isn't the singleflight from `go/x/sync`.
On the other hand, it still does the same thing but with context handling, so it's probably fine not to disambiguate them.